qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -103,22 +103,37 @@
         - @default:
             cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=2048 oflag=direct'
             cmd_md5 = 'md5sum %s'
+            folder_test = 'yes'
             test_folder = '%s/test_folder'
             cmd_new_folder = "mkdir -p ${test_folder}"
             cmd_copy_file = "cp %s ${test_folder}"
-            cmd_del_folder = "rm -rf ${test_folder}"
+            cmd_rename_folder = "mv ${test_folder} ${test_folder}_tmp"
+            cmd_del_folder = "rm -rf ${test_folder}_tmp"
+            cmd_check_folder = "ls ${test_folder}_tmp"
             io_timeout = 120
             Windows:
                 cmd_dd = 'dd if=/dev/random of=%s bs=1M count=200'
                 cmd_md5 = "%s: && md5sum.exe %s"
                 test_folder = '%s\test_folder'
                 cmd_new_folder = "md ${test_folder}"
                 cmd_copy_file = "copy %s ${test_folder}"
-                cmd_del_folder = "rd /q /s ${test_folder}"
+                cmd_rename_folder = "%s && rename test_folder test_folder_tmp"
+                cmd_del_folder = "rd /q /s ${test_folder}_tmp"
+                cmd_check_folder = "dir ${test_folder}_tmp"
             default..with_cache.none:
                 io_timeout = 600
             variants:
                 - @default:
+                - with_nfs:
```

Comment:
Is there a difference between `- with_nfs_source` and `- with_nfs`?
Can we merge them into one case?

Reply:
Actually they are different test scenarios,the previous one is for basic file/folder test. The other one is for xfstest.

Reply:
Okay, but I think we still can do it, like:
```
    variants: # case about testing source type
        - with_local_source:
            ...
        - with_nfs_source:
            ...
    variants: # case about testing virtiofsd option
        - with_cache:
            ...
        - with_no_writeback:
            ...
    variants: # case about testing tool
        - @default:
            ...
            variants:
                - @default:
                - with_multi_fs_sources:
                    ...
        - run_stress:
            variants:
                - with_fio:
                    ...
                - with_pjdfstest:
                    ...
                - with_xfstest:
                    ...
```
This is maybe more readable.

Reply:
In fact, I also feel that they can be merged, one is local testing, the other is nfs backend test.
Similar to yhong's opinion, please feel free to share your opinion. 

`        - with_local_source:
            ...
        - with_nfs_source:
            variants:
                - @default:
                - with_xfstest
`